daemon/src/connection.rs

```diff
@@ -138,7 +138,7 @@ impl Actor {
 
         match msg {
             wire::MakerToTaker::Heartbeat => {
-                tracing::info!("got the heartbeat");
+                tracing::debug!("received a heartbeat message from maker");
```

Comment:
```suggestion
                tracing::trace!("received a heartbeat message from maker");
```

Let's do trace please. This is not important information and we run on debug by default. It heavily clutters the logs :)